java/arcs/core/analysis/InformationFlow.kt

```diff
@@ -498,6 +498,10 @@ class InformationFlow private constructor(
         (if (partial) setOf(prefix) else emptySet()) +
           (this as FieldType.ListOf).primitiveType.accessPathSelectors(prefix, partial)
       }
+      FieldType.Tag.Nullable -> {
+        (if (partial) setOf(prefix) else emptySet()) +
+          (this as FieldType.NullableOf).innerType.accessPathSelectors(prefix, partial)
```

Comment:
Can you update the unit tests for this file?

Reply:
Added a positive and a negative case using nullable to ensure that nullable fields are treated the same as non-nullable fields (for information flow)